docker-compose.yml

```diff
@@ -47,6 +47,8 @@ services:
     # We are technically dependent on mysql. However, we are dependent on the
     # jobserv to actually do db migrations. Therefore, we wait on the jobserv
     # instead of mysql.
+    tmpfs:
+      - /data/workers
```

Comment:
You separated the comment about the "wait-for" and the wait-for. I think you can just add these two line right after `command` and be good